daemon/emer-main.c

```diff
@@ -222,6 +238,8 @@ on_bus_acquired (GDBusConnection *system_bus,
                     G_CALLBACK (on_set_enabled), daemon);
   g_signal_connect (server, "handle-upload-events",
                     G_CALLBACK (on_upload_events), daemon);
+  g_signal_connect (server, "handle-reload-config",
+                    G_CALLBACK (on_reload_config), daemon);
```

Comment:
Should this be a privileged operation, ie. added to `on_authorize_method_check`? You could make an argument that calling this method has no observable effect if `/etc/machine-id` hasn't changed, so it does not need to be authorized.